Resources/Prototypes/_Sunrise/Entities/Objects/Weapons/Guns/HMGs/hmgs.yml

```
@@ -47,26 +69,93 @@
+      map: ["enum.GunVisualLayers.Mag"]
+  - type: MagazineVisuals
+    magState: mag
+    steps: 4
+    zeroVisible: false
+  - type: Appearance
+
+- type: entity
+  name: DL6902 machine gun
+  parent: BaseItem
+  id: WeaponDL6902PowerPack
+  categories: [ HideSpawnMenu ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)